### PR TITLE
Sgreenberg fix national districts

### PR DESCRIPTION
**Pull request by sgreenberg**

### CODE REVIEW

**Comment by sgreenberg** [comment]
Not ready to review yet. Let's get @bobg's pr (https://github.com/SisterDistrictTech/db-scripts/pull/14) merged first. With that merged, the diff against master for this PR will just be one commit.

File: SD_DB_Setup.sql [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
-
-use SisterDistrict_dev;
-#use SisterDistrict;
-
 CREATE TABLE national_districts
 	(id int(11) NOT NULL auto_increment primary key,
     state varchar(2) NOT NULL,
-    district int(3) NOT NULL,
+    district int(3),
     district_abbr varchar(6) NOT NULL,
```
Comment (nsudarsanam) [inline SD_DB_Setup.sql:5]: @LinneaHarts fyi

File: sdlib/national_districts.py [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+                         ("WI", 3, "WI-03"),
+                         ("WI", 4, "WI-04"),
+                         ("WI", 5, "WI-05"),
+                         ("WI", 6, "WI-06"),
+                         ("WI", 7, "WI-07"),
+                         ("WI", 8, "WI-08"),
+                         ("WV", 1, "WV-01"),
+                         ("WV", 2, "WV-02"),
+                         ("WV", 3, "WV-03"),
+                         ("WY", None, "WY-00")])
```
Comment (nsudarsanam) [inline sdlib/national_districts.py:455]: this is the only really big change from bob's PR right? the merge is going to get interesting :)

**Comment by sgreenberg** [comment]
@bobg, @nsudarsanam - I rebased on top of master. Take a look at the PR. Once you both approve, I'll merge. The changes to national_districts.py were made following the pattern that Bob established for pres_races.

File: sdlib/national_districts.py [hunk 3]
```diff
@@ -17,32 +10,446 @@ def populate(cur, reset=False):
...
-                # Don't let MySQL assign AUTO_INCREMENT ids to the
-                # rows, because we want them to start at 1
-                id = 1 + len(ndtuples)
-                district_abbr = '%s-%s' % (state, district)
-                ndtuples.append((id, state, district, district_abbr))
-
         if reset:
             cur.execute('DELETE FROM national_districts')
 
         sql = ('INSERT INTO national_districts ' +
```
Comment (nsudarsanam) [inline sdlib/national_districts.py:16]: we probably want a comment somewhere of how we are treating "at-large" districts?

**Review by nsudarsanam — APPROVED** [review]

File: sdlib/national_districts.py (hunk 3 above)
Comment (sgreenberg) [reply sdlib/national_districts.py:16]: Sure thing. I added it here - https://docs.google.com/document/d/1KyW6t-lnKBAEi_fQykHuJ2ZYfaFj_8gyNTrZmMnKFcQ/edit#heading=h.653ok8drsfcf. 

As long as we have a database document, that seems like the place for such explanation.

**Review by bobg — APPROVED** [review]

**Comment by bobg** [comment]
LGTM

**Comment by LinneaHarts** [comment]
Looks good to me

**Comment by sgreenberg** [comment]
got the approval from Linnea in slack.

**Pull request merged**